### PR TITLE
add ids for shop page elements

### PR DESCRIPTION
**Pull request by errnoh**
Adds `id` attributes for most elements on the store page, allowing easier integration (selecting elements based on class/id instead of nth child).

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -46,15 +46,15 @@ export function Layout() {
 									setActiveChar(character.id)
 								}}
 							>
-								<div className="char-button">
+								<div className="char-button" id="character">
```
Comment (danreeves) [inline src/components/Layout.tsx:49]: Technically, in HTML ids should be unique on the page so these should be classNames

Comment (errnoh) [reply src/components/Layout.tsx:49]: Oh, right. Not a frontend dev :sweat_smile: . I'll do a second pass a bit later.

Comment (danreeves) [reply src/components/Layout.tsx:49]: No problem!

**Comment by errnoh** [comment]
Closing this one for now. I'll want to do proper pass on classes and ids at later point but we have quite many new features coming in, better to do separate refactoring / minor improvements sprint at later time.

**Pull request closed without merging**